src/server/courseInstance/courseInstance.module.ts

```diff
@@ -11,12 +11,18 @@ import { CourseInstanceService } from './courseInstance.service';
 import { CourseInstanceController } from './courseInstance.controller';
 import { ScheduleBlockView } from './ScheduleBlockView.entity';
 import { ScheduleEntryView } from './ScheduleEntryView.entity';
+import { CourseInstance } from './courseinstance.entity';
+import { Faculty } from '../faculty/faculty.entity';
+import { FacultyCourseInstance } from './facultycourseinstance.entity';
 
 @Module({
   imports: [
     TypeOrmModule.forFeature([
       Course,
+      CourseInstance,
       CourseListingView,
+      Faculty,
```

Comment:
Unless I'm misunderstanding, wouldn't the point of importing the `FacultyModule` be to get access to the `FacultyService`, `FacultyController`, etc.? If so I don't think that's necessary here as we're only using the `Faculty` repository directly...

Reply:
So, importing `FacultyModule` would also provide `Repository<Faculty>` to the `CourseInstanceModule`, rather than `CourseInstanceModule` re-generating `Repository<Faculty>` itself and having a direct dependency on the `Faculty` entity.

Reply:
_Recapping a discussion on Slack:_

_The proposed change is intended to modularize the different domains of the app, keeping the `Faculty` stuff in the `FacultyModule`, the `Course` stuff in the `CourseModule`, etc. This is something we should be doing, but setting that up is going to require some changes in the `FacultyModule` and, ideally, we'd make all of our modules consistent to that._

_Since the current implementation is working, we're going to leave it as is, and we'll open a new ticket to address those changes when we have time to handle refactoring and technical debt._

Reply:
Opened #408 to address this (and other instances of the same) later on